README.md

```diff
@@ -49,9 +46,18 @@ sudo ./spectre-meltdown-checker.sh
 
 ### Run the script in a docker container
 
+#### With docker-compose
+
+```shell
+docker-compose build
+docker-compose run --rm spectre-meltdown-checker
+```
+
+#### Without docker-compose
+
 ```shell
 docker build -t spectre-meltdown-checker .
-docker run --rm --privileged -v /boot:/boot:ro -v /lib/modules:/lib/modules:ro -v /dev/cpu:/dev/cpu:ro spectre-meltdown-checker
+docker run --rm --privileged -v /boot:/boot:ro -v /dev/cpu:/dev/cpu:ro -v /lib/modules:/lib/modules:ro spectre-meltdown-checker
```

Comment:
I try to always keep the diffs to a bare minimum, the inversion of parameters here doesn't seem necessary AFAICT?